Community Choice 2018 Results newspost (#2012)
ppy/osu-wiki: news/2019-02-04-community-choice-2018-results.md

---
layout: post
title: "Community Choice 2018: Results"
date: 2019-02-04 10:30:00 +0000
---

You've had your say, and we have our winners! Check out the new Community Favourites of 2018, featuring some of the closest voting we've ever seen!

[![](/wiki/shared/news/2019-01-17-community-choice-2018/header.jpg)](https://osu.ppy.sh/community/contests/74)

### osu!

[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/kda.jpg)](https://osu.ppy.sh/beatmapsets/874948)

> **2489** voters could feel the rush now as they enjoyed [Nathan](https://osu.ppy.sh/users/4785223)'s take on the video-game cross k-pop masterpiece that took the world by storm, **[K/DA - POP/STARS (ft. Madison Beer, (G)I-DLE, Jaira Burns)](https://osu.ppy.sh/beatmapsets/874948)**!

Reviewer: ```suggestion
**2489** voters could feel the rush now as they enjoyed [Nathan](https://osu.ppy.sh/users/4785223)'s take on the video-game cross K-pop masterpiece that took the world by storm, **[K/DA - POP/STARS (ft. Madison Beer, (G)I-DLE, Jaira Burns)](https://osu.ppy.sh/beatmapsets/874948)**!
```
osu nitpick tutorial

> This will likely come as a surprise to some who expected one map from the ubiquitous deluge of 'farming' maps to nab top place, but as this goes to show, sometimes you just need to stop farming creeps and get out there and start nabbing kills, something that Nathan and his collaborators managed with effortless ease.

Reviewer: ```suggestion
This will likely come as a surprise to some who expected one map from the ubiquitous deluge of "farm" maps to nab top place, but as this goes to show, sometimes you just need to stop farming creeps and get out there and start nabbing kills, something that Nathan and his collaborators managed with effortless ease.
```

The finalists were:

> [Panda Eyes - ILY by M a r v o l l o](https://osu.ppy.sh/beatmapsets/653534) with **1935** votes total.

Reviewer: for these I think you should only link the artist - title and then link user separately

Reviewer: also would be better formatted as a list (remove ending periods in that case)

Author: list formatting makes sense but i can't be bothered splicing links in for 40 users when the beatmapset listing takes them to the place anyway if they want to check the user at

[Vickeblanka - Black Rover (TV Size) by Sotarks](https://osu.ppy.sh/beatmapsets/781509) with **1911** votes total.

[DECO*27 - HIBANA feat. Hatsune Miku by Pho](https://osu.ppy.sh/beatmapsets/671199) with **1724** votes total.

[SOOOO - Happppy song by Kuron-kun](https://osu.ppy.sh/beatmapsets/744593) with **1713** votes total.

[Aitsuki Nakuru - Monochrome Butterfly by Sotarks](https://osu.ppy.sh/beatmapsets/770300) with **1679** votes total.

[Omoi - Teo by Kroytz](https://osu.ppy.sh/beatmapsets/721804) with **1572** votes total.

[Camellia feat. Nanahira - Bassdrop Freaks (2018 "Redrop" ver.) by Mir](https://osu.ppy.sh/beatmapsets/806859) with **1569** votes total.

[Camellia - Exit This Earth's Atomosphere (Camellia's ''PLANETARY//200STEP'' Remix) by ProfessionalBox](https://osu.ppy.sh/beatmapsets/855677) with **1475** votes total.

[YUC'e - Future Candy by Nathan](https://osu.ppy.sh/beatmapsets/546820) with **1388** votes total.

[Check out the full results over on the contest listing for this mode!](https://osu.ppy.sh/community/contests/74)

### osu!taiko

[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/xinobikill.jpg)](https://osu.ppy.sh/beatmapsets/742966)

> **121** drum fanatics stepped forward and said 'hell yes' to [SOUND HOLIC feat. Nana Takahashi - XINOBIKILL](https://osu.ppy.sh/beatmapsets/742966) by [Skull Kid](https://osu.ppy.sh/users/3044264), in some of the closest voting we've ever observed in a Community Choice poll OR a 'Best Of' to date.

Reviewer: ```suggestion
**121** drum fanatics stepped forward and said "hell yes" to [SOUND HOLIC feat. Nana Takahashi - XINOBIKILL](https://osu.ppy.sh/beatmapsets/742966) by [Skull Kid](https://osu.ppy.sh/users/3044264), in some of the closest voting we've ever observed in a Community Choice poll *or* a "Best Of" to date.
```

> **121** drum fanatics stepped forward and said "hell yes" to [SOUND HOLIC feat. Nana Takahashi - XINOBIKILL](https://osu.ppy.sh/beatmapsets/742966) by [Skull Kid](https://osu.ppy.sh/users/3044264), in some of the closest voting we've ever observed in a Community Choice poll *or* a "Best Of" to date.

Reviewer 2: Map link should be bold like osu's number one? Same goes for catch/mania. 
Makes the winning map name easier to notice.

Author: done

> Seriously, at some points, there was less than 3 votes splitting first, second, and third place. When we tweeted about the votes drawing to a close, there was a two-way tie for first place for at least a day before the votes were slated to close.

Reviewer: ```suggestion
Seriously, at some points, there were less than 3 votes splitting first, second, and third place. When we tweeted about the votes drawing to a close, there was a two-way tie for first place for at least a day before the votes were slated to close.
```

Yowza.

The finalists were:

[CustomiZ - COOLEST by Ascendance](https://osu.ppy.sh/beatmapsets/502512) with **139** votes total.

[Camellia - Ultimate Ascension by Deif](https://osu.ppy.sh/beatmapsets/520301) with **138** votes total.

[Chito (CV: Minase Inori), Yuuri (CV: Kubo Yurika) - More One Night (Assertive Hardcore Bootleg) \[short ver.\] by Ascendance](https://osu.ppy.sh/beatmapsets/781081) with **136** votes total.

[Icon For Hire - Make a Move (Speed Up Ver.) by Sotarks](https://osu.ppy.sh/beatmapsets/765778) with **134** votes total.

[LeaF - Alice in Misanthrope -Ensei Alice- by Deif](https://osu.ppy.sh/beatmapsets/797236) with **113** votes total.

[Function Phantom - Algebra by Spectator](https://osu.ppy.sh/beatmapsets/468281) with **107** votes total.

[dj TAKA - quaver by Ascendance](https://osu.ppy.sh/beatmapsets/879085) with **102** votes total.

[LiSA - ADAMAS (TV Size) by Ascendance](https://osu.ppy.sh/beatmapsets/867287) with **100** votes total.

> [Zektbach - Turii \~Panta rhei\~ by Kyuare](https://osu.ppy.sh/beatmapsets/685428) with **99** votes total.

Reviewer 3: panta rhei is still crossed out

Reviewer: you must be looking at an old revision, it's corrected now

[Check out the full results over on the contest listing for this mode!](https://osu.ppy.sh/community/contests/76)

### osu!mania

[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/mazare.jpg)](https://osu.ppy.sh/beatmapsets/650415)

**621** voters saw reason to celebrate [MAZARE - Mazare Party](https://osu.ppy.sh/beatmapsets/650415) by [Insp1r3](https://osu.ppy.sh/users/7131254) with a positively dominating lead over its competition, leading the pack by nearly a solid third of the next highest finalist.

> Having featured in the Seasonal Spotlights already, it's no surprise that this map was a crowd favorite.

Reviewer: ```suggestion
Having been featured in the Seasonal Spotlights already, it's no surprise that this map was a crowd favourite.
```

---
layout: post
title: "Community Choice 2018: Results"
date: 2019-02-04 10:30:00 +0000
---

You've had your say, and we have our winners! Check out the new Community Favourites of 2018, featuring some of the closest voting we've ever seen!

[![](/wiki/shared/news/2019-01-17-community-choice-2018/header.jpg)](https://osu.ppy.sh/community/contests/74)

### osu!

[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/kda.jpg)](https://osu.ppy.sh/beatmapsets/874948)

**2489** voters could feel the rush now as they enjoyed [Nathan](https://osu.ppy.sh/users/4785223)'s take on the video-game cross K-pop masterpiece that took the world by storm, **[K/DA - POP/STARS (ft. Madison Beer, (G)I-DLE, Jaira Burns)](https://osu.ppy.sh/beatmapsets/874948)**!

> This will likely come as a surprise to some who expected one map from the ubiquitous deluge of "farm" maps to nab top place, but as this goes to show, sometimes you just need to stop farming creeps and get out there and start nabbing kills, something that Nathan and his collaborators managed with effortless ease.

Reviewer 4: Is it intentional to repeat "farm" and "nab" here? Variety would be nice. Also I get that you were linking in with league r.e. the "kills" comment, but it makes no sense in this context because you're comparing it to votes... Do people die when they vote??

* [Unknown Artist - Miniministop Hitoyasumi no Uta\[Mokuyoubi\]Gakusei no Uta by Chromoxx](https://osu.ppy.sh/beatmapsets/705423) with **74** votes total.

* [Vickeblanka - Black Rover (TV Size) by Sotarks](https://osu.ppy.sh/beatmapsets/781509) with **72** votes total.

* [ESTi - HELIX (Edit ver.) by hikikochan](https://osu.ppy.sh/beatmapsets/625729) with **71** votes total.

* [ClariS - SHIORI vs. Hitorigoto by Sotarks](https://osu.ppy.sh/beatmapsets/707380) with **67** votes total.

* [XXXTENTACION - Jocelyn Flores by Chromoxx](https://osu.ppy.sh/beatmapsets/705133) with **66** votes total.

* [Will Stetson - Harumachi Clover (Swing Arrangement) [Dictate Edit] by Sotarks](https://osu.ppy.sh/beatmapsets/842412) with **63** votes total.

* [KASAI HARCORES - Cycle Hit by Axer](https://osu.ppy.sh/beatmapsets/668239) with **63** votes total.

[Check out the full results over on the contest listing for this mode!](https://osu.ppy.sh/community/contests/75)

### osu!catch

[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/asphyxia.jpg)](https://osu.ppy.sh/beatmapsets/758101)

> With **182** votes, the collaborative sets by [Ascendance](https://osu.ppy.sh/users/2931883), [Spectator](https://osu.ppy.sh/users/702598) and [Hareimu](https://osu.ppy.sh/users/4138746) on **[Co shu Nie - asphyxia (TV edit)](https://osu.ppy.sh/beatmapsets/758101)** hosted by [Monstrata](https://osu.ppy.sh/users/2706438) strangled out the competition solidly with a nearly 50 point lead.

Reviewer 4: "Sets" should probably be "difficulties" here to avoid confusion, since only one "set" can win. Also, a comma before and after "hosted by Monstrata" would probably make this read a little nicer.

Similar to osu!taiko, the voting for osu!catch was a super close endeavour all up until the last three or four days of voting, when asphyxia surged forward with a lead that the other sets could just not recover from.

> The agonizing choice many osu!catch players were forced to make is super obvious in the sheer vote gap between many of the finalist entries, shown purely in the fact that 2nd and 3rd place are literally one vote apart. 

Reviewer 4: "were" literally one vote apart? Past tense consistency in this sentence probably

The finalists were:

* [M2U - Marigold (feat. Guriri) by DE-CADE](https://osu.ppy.sh/beatmapsets/636272) with **417** votes total.

* [Billx - Equinox by Janko](https://osu.ppy.sh/beatmapsets/692910) with **412** votes total.

* [xi - Glorious Crown by Kuo Kyoka](https://osu.ppy.sh/beatmapsets/474900) with **401** votes total.

* [Getty vs. DJ DiA - DropZ-Line- by Fresh Chicken](https://osu.ppy.sh/beatmapsets/701369) with **348** votes total.

* [Kobaryo - Dotabata Animation [feat. t+pazolite] by YaHao](https://osu.ppy.sh/beatmapsets/573280) with **339** votes total.

* [jioyi - cyanine by Rivals_7](https://osu.ppy.sh/beatmapsets/581729) with **336** votes total.

* [Chroma - I by Lude](https://osu.ppy.sh/beatmapsets/738673) with **331** votes total.

* [Helblinde - Heaven's Fall by riunosk](https://osu.ppy.sh/beatmapsets/507745) with **290** votes total.

* [Ice vs. Morimori Atsushi - RE:UNION -Duo Blade Against- by Critical_Star](https://osu.ppy.sh/beatmapsets/734357) with **288** votes total.

[Check out the full results over on the contest listing for this mode!](https://osu.ppy.sh/community/contests/77)

--------------

> The winners for each mode will be taking home **a shiny new profile badge, and 6 months of osu!supporter** for their hard-earned win, and each finalist will also score **3 months of osu!supporter** for each win, stacking if they've had multiple maps featured in the voting.

Reviewer 4: for each "top 10 placement"? Only one set can actually "win" after all
